test/integration/edge_connectivity/edge_connectivity_test.go

```diff
@@ -0,0 +1,421 @@
+				},
+				{
+					from: "test-4-edge",
+					to:   "test-4-interior-1",
+				},
+				{
+					from: "test-4-edge",
+					to:   "test-4-interior-2",
+				},
+				{
+					from: "test-4-edge",
+					to:   "test-4-interior-3",
+				},
+			},
+			direct:   3,
+			indirect: 0,
+		},
+		// Test 5 ---------------------------------------------------
+		{
+			name: "test-5",
+			diagram: []string{"edge  -->  interior-1",
+				"interior-1  -->  interior-2"},
+			namespaces: []*SkupperNamespace{
+				&SkupperNamespace{
+					name:   "test-5-edge",
+					isEdge: true,
+				},
+				&SkupperNamespace{
+					name:   "test-5-interior-1",
+					isEdge: false,
+				},
+				&SkupperNamespace{
+					name:   "test-5-interior-2",
+					isEdge: false,
+				},
+			},
+			connexions: []Connexion{
+				{
+					from: "test-5-edge",
+					to:   "test-5-interior-1",
+				},
+				{
+					from: "test-5-interior-1",
+					to:   "test-5-interior-2",
+				},
+			},
+			direct:   1,
+			indirect: 1,
+		},
+	}
+
+	//--------------------------------------------
+	// Run the tests.
+	//--------------------------------------------
+	for _, test := range tests {
+		ctx, cancel := context.WithCancel(context.Background())
+		defer cancel()
+
+		var err error
+
+                /*
+		if !*clusterRun {
+			t.Skip(fmt.Sprintf("%sSkipping: This test only works in real clusters.%s", red, resetColor))
+			return
+		}
+                */
+
+		if verbose {
+			fp(os.Stdout, "\n\n%s%s\n\n", cyan, test.name)
+			for _, s := range test.diagram {
+				fp(os.Stdout, "\t%s\n", s)
+			}
+			fp(os.Stdout, "\n\tdirect: %d   indirect: %d\n", test.direct, test.indirect)
+			fp(os.Stdout, "%s\n\n", resetColor)
+		}
+
+		// Here is where we will put the connection tokens, for
+		// any of the namespaces in this test that require them.
+		testPath := "./tmp/"
+		os.Mkdir(testPath, 0755)
+
+		// Before we go though the list of namespaces to create them,
+		// make a set of all namespace that will need to create connexion tokens.
+		needToken := make(map[string]bool, 0)
+		for _, cnx := range test.connexions {
+			needToken[cnx.to] = true
+		}
+
+		//-----------------------------------------------------
+		// Create and populate each namespace.
+		//-----------------------------------------------------
+		for _, namespace := range test.namespaces {
+			namespace.client, err = cli.NewClient(namespace.name, "", "")
+			check(t, err, test.name, fmt.Sprintf("Can't create client for namespace |%s|.", namespace.name))
+			info(test.name, fmt.Sprintf("Created client for namespace |%s|.", namespace.name))
+
+			_, err = kube.NewNamespace(namespace.name, namespace.client.KubeClient)
+			defer kube.DeleteNamespace(namespace.name, namespace.client.KubeClient)
+			check(t, err, test.name, fmt.Sprintf("Can't create namespace |%s|.", namespace.name))
+			info(test.name, fmt.Sprintf("Created namespace |%s|.", namespace.name))
+
+			err = namespace.client.VanRouterCreate(ctx, types.VanSiteConfig{
```

Comment:
You should rebase this pr on master. Van prefix has been removed from master. And! the rebase wil show no conflict, butt master will be bronken, you have to manually remove all "Van" prefix in this file (just a few) in order to fix that.

Reply:
done.